source/Octopus.Client.Tests/PublicSurfaceAreaFixture.ThePublicSurfaceAreaShouldNotRegress..NETCore.approved.txt

```diff
@@ -6618,7 +6618,7 @@ Octopus.Client.Repositories
   interface IDashboardRepository
   {
     Octopus.Client.Model.DashboardResource GetDashboard()
-    Octopus.Client.Model.DashboardResource GetDynamicDashboard(String[], String[])
+    Octopus.Client.Model.DashboardResource GetDynamicDashboard(String[], String[], Boolean)

```

Comment:
What happens to customers who are currently using `GetDynamicDashboard`?

Reply:
Ah yes! good point. um would an overload method be solution to that?

Reply:
Yes, I think an overloaded method would be a good solution or a default value in the interface.